### DISCUSSION TITLE
Run multiple instances of neovide in the same app (just like having tabs/buffers)

### DISCUSSION THREAD

**Discussion by idelice** [neovide/neovide#2365] [Ideas]
It becomes very frustrating when you have multiple neovide "apps" open and they all look the same, and in order to fix that, I suggest that you make it possible to having tabs/buffers where you can switch between the multiple instances.

And, also, having the option to switch to another project - inside the neovide - would be such a great feature.

**Comment by mrded** [comment]
Here is a related issue: https://github.com/neovide/neovide/issues/2378
